s3/Acornfile

```diff
@@ -83,8 +95,13 @@ jobs: {
 	apply: {
 		dependsOn: ["cleanup"]
 		build: {
-			context:    "."
-			dockerfile: "Dockerfile"
+			context: "."
+			if args.dryRun {
+				dockerfile: "test.Dockerfile"
+			}
```

Comment:
There is an else now. 
```
dockerfile: if args.dryRun { "test.Dockerfile" } else {"Dockerfile"}
```

Reply:
Would prefer we do build targets in a single file.

Reply:
It's a single Dockerfile with no conditionals now